xml/libvirt_migrating_vms.xml

```diff
@@ -0,0 +1,623 @@
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><option>--undefinesource</option></term>
+        <listitem>
+          <para>
+            When specified, the &vmguest; definition on the source host is
+            deleted after a successful migration (however, virtual disks
+            attached to this guest are <emphasis>not</emphasis> deleted).
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><option>--parallel --parallel-connections <replaceable>NUM_OF_CONNECTIONS</replaceable></option></term>
+        <listitem>
+          <para>
+            Parallel migration can be used to increase migration data throughput
+            in cases where a single migration thread is not capable of
+            saturating the network link between source and target hosts. On
+            hosts with 40&nbsp;GB network interfaces, it may require four
+            migration threads to saturate the link. With parallel migration, the
+            time required to migrate large memory VMs can be reduced.
+          </para>
+        </listitem>
+      </varlistentry>
+    </variablelist>
+
+    <para>
+      The following examples use &wsIVname; as the source system and &wsIname;
+      as the target system; the &vmguest;'s name is
+      <literal>opensuse131</literal> with ID <literal>37</literal>.
+    </para>
+
+    <variablelist>
+      <varlistentry>
+        <term>Non-live migration with default parameters</term>
+        <listitem>
+<screen>&prompt.user;virsh migrate 37 qemu+ssh://&exampleuser_plain;@&wsIname;/system</screen>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Transient live migration with default parameters</term>
+        <listitem>
+<screen>&prompt.user;virsh migrate --live opensuse131 qemu+ssh://&exampleuser_plain;@&wsIname;/system</screen>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Persistent live migration; delete VM definition on source</term>
+        <listitem>
+<screen>&prompt.user;virsh migrate --live --persistent --undefinesource 37 \
+qemu+tls://&exampleuser_plain;@&wsIname;/system</screen>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Non-live migration using port 49152</term>
+        <listitem>
+<screen>&prompt.user;virsh migrate opensuse131 qemu+ssh://&exampleuser_plain;@&wsIname;/system \
+--migrateuri tcp://@&wsIname;:49152</screen>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>Live migration transferring all used storage</term>
+        <listitem>
+<screen>&prompt.user;virsh migrate --offline --persistent --copy-storage-all \
+opensuse156 qemu+ssh://&exampleuser_plain;@&wsIname;/system</screen>
+          <important>
+            <para>
+              All VM's storage must be placed in a &libvirt; storage pool. The
```

Comment:
We should clearly state that the use of libvirt storage pools are only required when migrating the VM' storage. I.e. when using the --copy-storage-all option. Storage pools are not required in the typical migration scenario where shared storage is used.